src/domain/checks/GetPaymentNotificationMetadataChecks.ts

```diff
@@ -26,7 +28,27 @@ export const hasPAGOPAAsAttachmentNameC = flow(
   RA.exists(({ input }) => input.attachmentName === 'PAGOPA')
 );
 
+const calledDownloadEndpointC = pipe(
```

Comment:
WDYT about trying to unify the two `calledDownloadEndpointC`?

Reply:
I totally agree with you. I'm going to take a look at it and try to unify them!

Reply:
Pushed some refactor